graphium/cli/finetune_utils.py

```diff
@@ -70,9 +63,70 @@ def benchmark_tdc_admet_cli(save_dir, wandb, name, inclusive_filter):
         ret = {k: v.item() for k, v in ret.items()}
         results[n] = ret
 
+    # Save to the results_dir by default or to the Hydra output_dir if needed.
+    # This distinction is needed, because Hydra's output_dir cannot be remote.
+    save_dir = cfg["constants"].get("results_dir", HydraConfig.get()["runtime"]["output_dir"])
     fs.mkdir(save_dir, exist_ok=True)
     path = fs.join(save_dir, "results.yaml")
     logger.info(f"Saving results to {path}")
 
     with fsspec.open(path, "w") as f:
         yaml.dump(results, f)
+
+
+@finetune_app.command(name="fp")
```

Comment:
is `fp` well known? why not `fingerprint` or `feature_vector`?

Reply:
I think it is well known, but I'm in favor of making it more explicit! 👍

Reply:
More explicit please :)